sympy/printing/tests/test_repr.py

```diff
@@ -101,12 +101,11 @@ def test_Rational():
 
 
 def test_Float():
-    sT(Float('1.23', prec=3), "Float('1.22998', prec=3)")
```

Comment:
`srepr` is used just to yield the string representation of the given Float, not for any further computation. I don't see why one would want to see extra decimal digits as compared to imput.